### ISSUE TITLE
Govcloud support (AWS)

### ISSUE THREAD

**Issue by rirze** [snyk/driftctl#1339]
**Description**

`driftctl` does not appear to support AWS GovCloud regions.

**Environment**
 - OS: Mac OS
 - driftctl version: latest
 - terraform version:
 - terraform providers versions: 

**How to reproduce**

Run `driftctl` against an account in a GovCloud region (e.g. `us-gov-west-1`).

**Possible Solution**

I willing to provide support to enable `driftctl` working in GovCloud regions.

**Additional context**

The first error I run into is:
```
Scanned states (1) 
error scanning resource type aws_cloudfront_distribution: RequestError: send request failed
caused by: Get "https://cloudfront.us-gov-west-1.amazonaws.com/2020-05-31/distribution": dial tcp: lookup cloudfront.us-gov-west-1.amazonaws.com on 75.75.75.75:53: no such host
```
Which makes sense, since there is no CloudFront service in GovCloud.

**Comment by sjourdan** [comment]
Hi! Thanks for reporting this. 

1) we don't know any way to test on gov cloud, would you be up to testing alpha builds for us? 

2) can you elaborate on this 75.75.75.75 dns resolution error? Is this a common DNS resolution on AWS govcloud? 

Cheers!

**Comment by rirze** [comment]
@sjourdan 

1. Sure, I'd be open to testing alpha builds.
2. Put simply, I don't know about the DNS error specifically. Just that Cloudfront doesn't exist in the AWS Govcloud space. So in essence, that url will never work for GovCloud.

**Comment by wbeuil** [comment]
@rirze thanks for opening this issue. You could potentially use driftctl without scanning specific resources like `aws_cloudfront_distribution` with a driftignore file at the root of your repo. Did you already try?

```
aws_cloudfront_distribution.*
```

**Comment by rirze** [comment]
> @rirze thanks for opening this issue. You could potentially use driftctl without scanning specific resources like `aws_cloudfront_distribution` with a driftignore file at the root of your repo. Can you try?
> 
> ```
> aws_cloudfront_distribution.*
> ```

Thanks, I'll try that. I had previously set a filter for `aws_security_group` at the command line, but ran into this error. 

To clarify, if I specify a filter at the command line, will `driftctl` still scan all resources? In that case, is that command option more of a post-scan filter?

**Comment by rirze** [comment]
@wbeuil I was able to get a relevant report per my needs following your advice. I put this in my `.driftignore`:
```
*
!aws_security_group*
```
And I was able to get the current state of my Terraform migration wrt to Security Group and their rules. 
I guess my approach for now will be to slowly add resources paths that I care about into the `.driftignore` until I run into more issues. 

Feel free to close this for now-- I can open up specific issues down the road if I run into any.
